base/applications/regedit/framewnd.c

```diff
@@ -1382,7 +1377,7 @@ static BOOL _CmdWndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
         RefreshListView(g_pChildWnd->hListWnd, hKeyRoot, keyPath, TRUE);
         break;
         /*case ID_OPTIONS_TOOLBAR:*/
-        /*	toggle_child(hWnd, LOWORD(wParam), hToolBar);*/
+        /* toggle_child(hWnd, LOWORD(wParam), hToolBar);*/
         /*    break;*/
```

Comment:
If we're here to make indentation consistent (even in a commented out code), it should be at the same level as "break".

Reply:
ok